core-test/src/test/scala/org/bitcoins/core/protocol/tlv/EventDescriptorTest.scala

```diff
@@ -0,0 +1,39 @@
+package org.bitcoins.core.protocol.tlv
+
+import org.bitcoins.core.number.{Int32, UInt16, UInt32}
+import org.bitcoins.testkit.util.BitcoinSUnitTest
+
+class EventDescriptorTest extends BitcoinSUnitTest {
+
+  behavior of "EventDescriptor"
+
+  it must "create an enumerated event" in {
+    val outcomes = Vector("Democrat_win", "Republican_win", "other")
+    val enumEventDescriptorV0TLV = EnumEventDescriptorV0TLV(outcomes)
+
+    assert(enumEventDescriptorV0TLV.noncesNeeded == 1)
+  }
+
+  it must "create a range event" in {
+    val rangeEventDescriptorV0TLV =
+      RangeEventDescriptorV0TLV(start = Int32(-2),
+                                count = UInt32(4),
+                                step = UInt16.one,
+                                unit = "",
+                                precision = Int32.one)
+
+    assert(rangeEventDescriptorV0TLV.outcomes == Vector("-2", "-1", "0", "1"))
+  }
```

Comment:
Since these are `precision = 1` doesn't that mean they should all by x10

Reply:
I don't know, this is the formula you gave me. There is no formula in the docs 

https://github.com/discreetlogcontracts/dlcspecs/blob/540c23a3e89c886814145cf16edfd48421d0175b/Oracle.md#range

https://suredbits.slack.com/archives/CB1CB4AUX/p1604013798441800

Reply:
The precision is not used. Why do we have precision here?